src/components/splash_screen.js

```diff
@@ -1,44 +1,84 @@
-import React from "react"
+import React, { useEffect, useState } from "react"
+import PropTypes from "prop-types"
+import { disablePageScroll, enablePageScroll } from "scroll-lock"
+import uuid from "uuid/v4"
```

Comment:
I don't think you should import this given that we already have `lodash/uniqueId` and it performs the same function and we don't need to fill in extra bundle size :+1:

Reply:
Nice, didn't know about `_.uniqueId`. Seems like I have a new optimization to do :stuck_out_tongue: